admin/agent-management.rst

```diff
@@ -396,12 +398,12 @@ undeployed to node - no agent restart is required.
 File delivery policy
 --------------------
 
-File delivery policy is created to automatically upload files form server to agents.
+File delivery policy is created to automatically upload files from server to agents.
 
-First root folder or folders should be created - folders with the full path to place
-where uploaded file and folder structure should be placed. After folder
-structure is created files can be added to this structure. On policy apply folders will be
-created if possible and files will be uploaded.
+Firstly, :ref:`root folder<agent-policy-file>` or folders should be created - folders with the full path to location

```

Comment:
Do not understand why there is reference to new policy creation. Root folder should be created already inside new policy - it is not new policy itself.